### PR TITLE
change build machine from build08 to build05 for ovn-kube

### PR DESCRIPTION
**Pull request by JacobTanenbaum**
this is the output of make jobs that is needed since https://github.com/openshift/release/pull/43639 was merged

### CODE REVIEW

**Review by tssurya — APPROVED** [review]
/lgtm

**Comment by droslean** [comment]
/hold

Temporary holding to unblock the queue

**Pull request closed without merging**
